cvat/apps/events/const.py

```diff
@@ -0,0 +1,10 @@
+# Copyright (C) 2024 CVAT.ai Corporation
+#
+# SPDX-License-Identifier: MIT
+
+import datetime
+
+MAX_EVENT_DURATION = datetime.timedelta(seconds=100)
+WORKING_TIME_RESOLUTION = datetime.timedelta(milliseconds=1)
+WORKING_TIME_SCOPE = 'send:working_time'
+COLLAPSED_EVENT_SCOPES = frozenset(("change:frame",))
```

Comment:
Probably, it makes sense to rename either this constant or the `compressed_*` tests to use the same term. I'd vote for the constant renaming.